heppy/test/test_gun.py

```diff
@@ -18,7 +18,7 @@
 class TestAnalysis_gun(unittest.TestCase):
 
     def setUp(self):
-        random.seed(0xdeadbeef)
+        #random.seed(0xdeadbeef)
```

Comment:
This is necessary to get reproducible results. why this change?

Reply:
A test was failing for me because a temporary directory the test wanted to create using `mkdtemp` already existed.  I was just checking if that was the result of using the same seed, but I see now that it's unrelated. Will remove